# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/config_override.h
# Commit: 5147903580b8
# Full Hash: 5147903580b8f39197d67bddd5a0fb91d3670da4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-27 04:32:10
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r 0aad3393b572 -r 5147903580b8 media/ffvpx/config_override.h
--- a/media/ffvpx/config_override.h	Mon Feb 26 16:07:07 2024 +0000
+++ b/media/ffvpx/config_override.h	Mon Feb 26 16:07:08 2024 +0000
@@ -1,11 +1,9 @@
 // This file contains overrides for config.h, that can be platform-specific.
 
-#ifdef MOZ_LIBAV_FFT
 #undef CONFIG_FFT
 #undef CONFIG_RDFT
 #define CONFIG_FFT 1
 #define CONFIG_RDFT 1
-#endif
 
 // override '#define EXTERN_ASM _' in config_generic.h to allow building with
 // gcc on arm